### ISSUE TITLE
How to make terrestrial_virtual_channel_section .... descriptors_loop = [] not empty ?

### ISSUE THREAD

**Issue by mzeddd** [aventuri/opencaster#7]
Using this code it is not possible to add descriptors into TVCT table.

Do you have any plans to make it possible?
The same question about SST and RRT tables.

**Comment by aventuri** [comment]
heelo, you talking about ATSC stuff?  i fear we ar enot currently planning that extension..

**Comment by mzeddd** [comment]
Yes, it is ATSC stuff. I'm trying to make TS with TVCT which should have at least one descriptor.
Is it hard to add it?
Thanks in advance!
